src/index.js

```diff
@@ -537,7 +537,7 @@ class PhoneInput extends React.Component {
       formattedNumber
     }, () => {
       this.cursorToEnd();
-      if (this.props.onChange) this.props.onChange(formattedNumber.replace(/[^0-9]+/g,''), this.getCountryData());
```

Comment:
this regex replace was turning this into an unformatted number. Was that just an artifact from a previous implementation? Or was the intention something I am not seeing?

In handleInput it uses the formattedNumber https://github.com/bl00mber/react-phone-input-2/pull/205/files#diff-1fdf421c05c1140f6d71444ea2b27638L510